modin/core/storage_formats/base/query_compiler.py

```diff
@@ -1879,35 +1879,32 @@ def var(self, **kwargs):  # noqa: PR02
 
     # END Abstract column/row partitions reduce operations
 
-    # Abstract column/row partitions reduce operations over select indices
-    #
-    # These operations result in a reduced dimensionality of data.
-    # Currently, this means a Pandas Series will be returned, but in the future
-    # we will implement a Distributed Series, and this will be returned
-    # instead.
     @doc_utils.add_refer_to("DataFrame.describe")
-    def describe(self, **kwargs):  # noqa: PR02
+    def describe(
+        self,
+        percentiles: np.ndarray,
```

Comment:
Are we removing `include` + `exclude` here since we expect the DataFrame class to handle the arguments and pass in a masked QC?